docs/changes.rst

```diff
@@ -54,6 +54,9 @@ Released: 2023-01-14
   some operations (invoke, update) could cause data integrity issues).
   (see issue #2951)
 
+* Limited version of urllib3 used to lt 2.0 since there are incompatilities
+  in version 2.0.  (see issue #3001)
```

Comment:
If we consider this a temporary change, I suggest not to have a change log entry.

Reply:
This should still be done.

Reply:
Log entry removed